aws_codeseeder/_bundle.py

```diff
@@ -129,7 +129,10 @@ def generate_bundle(
     if files is not None:
         for src_file, name in files:
             LOGGER.debug(f"***file={src_file}:name={name}")
-            shutil.copy(src=src_file, dst=os.path.realpath(os.path.join(bundle_dir, name)))
+            dst_file = os.path.realpath(os.path.join(bundle_dir, name))
+            if "/" in name:
```

Comment:
How will this work on windows OS?

Reply:
paths on windows for python still need to be specified with `/`.